srcpkgs/drumstick/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'drumstick'
+pkgname=drumstick
+version=1.1.2
+revision=1
+build_style=cmake
+configure_args="-DLIB_SUFFIX="
+hostmakedepends="pkg-config doxygen libxslt docbook-xsl"
+makedepends="qt5-svg-devel alsa-lib-devel pulseaudio-devel"
+short_desc="a set of MIDI libraries using C++/Qt5 idioms and style"
+maintainer="Mohammad Amin Sameti <mamins1376@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://${pkgname}.sourceforge.net"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.bz2"
+checksum=7791b439aab9a5c24a53cbf5ce9ab1dcaf40a33a9d8828ca1731e823b1c7924e
+
+post_install() {
+	for doc in COPYING TODO INSTALL NEWS ChangeLog README AUTHORS; do
+		vdoc $doc
+	done
+
+	vlicense COPYING
+}
+
+drumstick-devel_package() {
+	short_desc+=" - development"
```

Comment:
` - development libraries`

Reply:
Actually there seems to be a need for a third, lib package; since the binaries themselves are also dependent on those. Not sure how to resolve that

Reply:
Just leave the libs in the main package?
It's bad form to have a main package depend on a subpackage.

Reply:
The main package will be `libdrumstick` and subpackages are `libdrumstick-devel` and `drumstick` which holds all the executable binaries, icons, man pages, etc.

Reply:
Why? Just make them all `drumstick` and `drumstick-devel.`

Reply:
Because it is also used as a lib dependency; e.g. by *vmpk* (coming next). If someone is going to only use *vmpk* and not *drumstick* binaries, then there is no need to keep them around.

Reply:
How large are the drumstick binaries?

Reply:
Each around 250K, with total of 1.8M. Of course, there are meta-files, which over all make a package of size 309k compressed.

Oops, seem like it's not worth it?

Reply:
Not worth splitting out small useful binaries.